homeassistant/components/weatherflow_cloud/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the WeatherflowCloud integration."""
+import logging
+
+DOMAIN = "weatherflow_cloud"
```

Comment:
Why isn't this just called `weatherflow`?

Reply:
Its in the description - but the TLDR is:

Weatherflow provides a UDP based integration completely local and a REST based integration that is cloud based - and uses additional sensors in the area to come up with data

Reply:
Thanks. Should perhaps have read that first 😁 👍